opencl/CMakeLists.txt

```diff
@@ -78,4 +78,24 @@ endif()
 FetchContent_MakeAvailable(ocl-icd)
 add_library(OpenCL-ICD ALIAS OpenCL)
 
+# OpenCL RT build requires headers from ICD loader, propagating location
+set(OpenCL_LOADER_DIR
+  ${ocl-icd_SOURCE_DIR} CACHE PATH "Path to OpenCL Loader")
+
```

Comment:
This is not needed. You should just link with OpenCL-ICD.

Reply:
The CMake file in OpenCL CPU RT uses this "OpenCL_LOADER_DIR" variable.

Reply:
I don't see any uses in this PR, but as I said in my previous comment OpenCL CPU RT should link with OpenCL-ICD instead of adding `OpenCL_LOADER_DIR` to additional include paths. 
See usage examples in the documentation: https://github.com/KhronosGroup/OpenCL-ICD-Loader#example-use.

Reply:
OCL CPU RT file includes "icd_dispatch.h" header file from OpenCL ICD Loader.  So we need to add OpenCL_LOADER_DIR to include path.

Reply:
It's added automatically by linking with the library.

Reply:
Fixed. thanks